app/models/analysis_parameter.rb

```diff
@@ -101,6 +101,22 @@ def value_method_type
     end
   end
 
+  # helper to format a user-supplied value based in parameter type
+  def formatted_user_value(value)
+    if value.blank?
+      ''
+    else
+      case self.parameter_type
+      when /String/
+        !value.start_with?('"') ? "\"#{value}\"" : value
```

Comment:
Can this be cleaner like so?
```suggestion
        !value.start_with?('"') ? '"#{value}"' : value
```